dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/tests/JakartaEE10Test.java

```diff
@@ -52,6 +52,7 @@ private static Set<String> getCompatFeatures(boolean openLibertyOnly) {
         compatFeatures.remove("jsonpContainer-2.1");
         compatFeatures.remove("passwordUtilities-1.1");
         compatFeatures.remove("persistenceContainer-3.1");
+        compatFeatures.remove("audit-2.0");//depends on restHandler-1.0
```

Comment:
```suggestion
        compatFeatures.remove("audit-2.0");
```
It doesn't depend on restHandler-1.0 so that does seem weird to make that statement